CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -1,23 +1,108 @@
 [global]
-includes = /mu2e/app/users/srsoleti/workDir/Production/CampaignConfig/mdc2020_prolog.cfg
-desc = MDC2020n
+includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020u/Production/CampaignConfig/mdc2020_prolog.cfg
+outdir_sim_nersc = /pnfs/mu2e/scratch/datasets/phy-sim/sim/mu2e/
+logdir_bck_nersc = /pnfs/mu2e/persistent/datasets/phy-etc/bck/mu2e/
+desc = MDC2020u_test_v2
+db_folder = mdc2020t
+run_number = 1205
 release = MDC2020
-release_v_o = n
+release_v_o = t
 njobs = 50
 
-[submit]
-tar_file_name = dropbox:///mu2e/data/users/srsoleti/museTarball/tmp.BIdJ4mQMgR/Code.tar.bz2
-use-cvmfs-dropbox = True
+[stage_extractednofield_fcl]
+global.stage_name = CosmicCORSIKAExtractedNoField
+job_output.filter_metadata = parents
 
-[job_setup]
-source_1 = /cvmfs/mu2e.opensciencegrid.org/setupmu2e-art.sh
-source_2 = ${CONDOR_DIR_INPUT}/Code/Code/setup.sh
+job_output.dest = https://dbdata0vm.fnal.gov:9443/mu2e_ucondb_prod/app/data/%(db_folder)s/
+job_setup.source_3 = $CONDOR_DIR_INPUT/ucondb_auth.sh
+job_setup.prescript = fileNamesGenerator.sh 0 %(njobs)s %(submitter)s > filenames.txt
+executable.name = true
+executable_2.name = generate_fcl
+executable_2.arg_4 = --inputs=filenames.txt
+executable_2.arg_5 = --merge-factor=1
+executable_2.arg_6 = --include
+executable_2.arg_7 = Production/JobConfig/cosmic/ExtractedNoFieldCORSIKA.fcl
+job_setup.ifdh_art = False
+submit.expected-lifetime = 71h
+
+[stage_extractednofield]
+global.stage_name = CosmicCORSIKAExtractedNoField
+
+job_output_2.add_to_dataset = dts.%(submitter)s.%(stage_name)s.%(desc)s.art
+job_output_2.dest = %(outdir_dts_tape)s/%(stage_name)s/%(desc)s/art/
+job_output_1.dest = %(logdir_bck)s/%(stage_name)s/%(desc)s/tbz/
+submit.memory = 4000MB
```

Comment:
Is 4000MB a well thought out number?

Reply:
I dont, currently we are working on optimizing these